frontend/src/stimulus/controllers/dynamic/work-packages/progress/preview.controller.ts

```diff
@@ -33,6 +33,12 @@ import { debounce, DebouncedFunc } from 'lodash';
 import Idiomorph from 'idiomorph/dist/idiomorph.cjs';
 import { FrameElement, TurboBeforeFrameRenderEvent } from '@hotwired/turbo';
 
+declare global {
+  interface HTMLElementTagNameMap {
+    'turbo-frame':FrameElement
+  }
+}
```

Comment:
There is an upstream PR for this: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71699